overlord/devicestate/handlers_install.go

```diff
@@ -350,6 +350,9 @@ func (m *DeviceManager) checkEncryption(st *state.State, deviceCtx snapstate.Dev
 			return false, fmt.Errorf("cannot encrypt device storage as mandated by encrypted storage-safety model option: %v", checkEncryptionErr)
 		}
 
+		// log any errors even if they are not fatal
+		logger.Noticef("ignoring error from check encryption: %v", checkEncryptionErr)
```

Comment:
maybe we want two different logs depending on hasFDESetupHook:

not encrypting device storage as querying kernel fde-setup hook did not succeed: %v

not encrypting device storage as checking TPM gave: %v

perhaps?

Reply:
agreed I think that would be nicer

Reply:
Thanks, I updated the code for this now.